client/components/TestRun/index.jsx

```diff
@@ -192,14 +209,98 @@ class TestRun extends Component {
                 </Helmet>
                 <Container fluid>
                     <Row>
-                        <Col>{heading}</Col>
+                        <aside className="col-md-3 test-navigator">
```

Comment:
@spectranaut - Do you know the best way with `aria` to markup the table of contents?

I think we might need something for the `aside` and/or `ol` I definitely want to add a way to represent the result state for each `li` which is currently on in the icon.  

Do if you don't "know" is there a recommended way you would look something like this up?